### ISSUE TITLE
Invalid URL warning for data urls

### ISSUE THREAD

**Issue by rahul-sekhar** [thoughtbot/capybara-webkit#712]
After upgrading to 1.4.0, in a test to do with image uploads, I get warnings saying:

```
Request to unknown URL: data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2w...
```

In the first place, I'm not even sure where these data URLs are being used. Regardless though, why would this be happening?

**Comment by rahul-sekhar** [comment]
Temporarily bypassed this by adding to the test:

```
page.driver.allow_url("")
```

**Comment by mhoran** [comment]
We introduced warnings for unknown remote URLs in capybara-webkit 1.4.0. Data URLs should probably be whitelisted, in addition to localhost (which is whitelisted by default). Thanks for the report! We'll get this fixed in 1.4.1.

**Comment by mhoran** [comment]
Fixed in 1.4.1.
